docs/wagtail-pages.md

```diff
@@ -0,0 +1,207 @@
+
+For example, in our `BrowsePage` (used in the [database fields example above](#fields),
+[the `secondary_nav_exclude_sibling_pages` `BooleanField` is added to the `sidefoot_panels` as a `FieldPanel`](https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/v1/models/browse_page.py#L63):
+
+```python
+from django.db import models
+from wagtail.wagtailadmin.edit_handlers import FieldPanel
+
+from v1.models.base import CFGOVPage
+
+
+class BrowsePage(CFGOVPage):
+    secondary_nav_exclude_sibling_pages = models.BooleanField(default=False)
+
+    # …
+
+    sidefoot_panels = CFGOVPage.sidefoot_panels + [
+        FieldPanel('secondary_nav_exclude_sibling_pages'),
+    ]
+```
+
+In out `LandingPage` (used in the [StreamFields example above](#streamfields), 
+[the `header` StreamField is added to the `content_panels` as a `StreamFieldPanel`](https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/v1/models/landing_page.py#L31):
+
+```python
+from wagtail.wagtailadmin.edit_handlers import StreamFieldPanel
+from wagtail.wagtailcore.fields import StreamField
+
+from v1.atomic_elements import molecules
+from v1.models import CFGOVPage
+
+
+class LandingPage(CFGOVPage):
+    header = StreamField([
+        ('hero', molecules.Hero()),
+        ('text_introduction', molecules.TextIntroduction()),
+    ], blank=True)
+
+    # …
+
+    content_panels = CFGOVPage.content_panels + [
+        StreamFieldPanel('header'),
+        # …
+    ]
+```
+
+## Parent / child page relationships
+
+Wagtail provides two attributes to page models that enable restricting the types of subpages or parent pages a particular page model can have. 
+
+For example, in our [interactive regulations page models](https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/regulations3k/models/pages.py#L138) 
+we have a `RegulationLandingPage` that can be created anywhere in the page tree. 
+`RegulationLandingPage`, however, can only have two types of pages created within it: 
+`RegulationPage` and `RegulationSearchPage`. 
+This parent/child relationship is expressed by setting `subpage_types` on `RegulationLandingPage`
+and `parent_page_types` on `RegulationPage` and `RegulationSearchPage` 
+to either the model classes themselves or to a model name in the form `app_label.ModelName`:
+
+```python
+from v1.models import CFGOVPage
+
+
+class RegulationLandingPage(CFGOVPage):
+    subpage_types = ['regulations3k.RegulationPage', 'RegulationsSearchPage']
+
+
+class RegulationsSearchPage(CFGOVPage):
+    parent_page_types = ['regulations3k.RegulationLandingPage']
+    subpage_types = []
+
+
+class RegulationPage(CFGOVPage):
+    parent_page_types = ['regulations3k.RegulationLandingPage']
+    subpage_types = []
+```
+
+!!! note 
+    Note: by setting `subpage_types` to an empty list on `RegulationPage` and `RegulationSearchPage`, 
+    we ensure that neither of these pages can have child pages.
```

Comment:
This wording is a little confusing. Perhaps instead "we prevent child pages from being added to these page types."